met-web/src/components/engagement/dashboard/comment/CommentViewContext.tsx

```diff
@@ -55,6 +56,27 @@ export const CommentViewProvider = ({ children }: { children: JSX.Element | JSX.
 
     const { page, size } = paginationOptions;
 
+    const validateComments = (commentsToValidate: Comment[]) => {
+        const hasApprovedComments = commentsToValidate.length > 0;
+
+        if (!hasApprovedComments) {
+            dispatch(
+                openNotificationModal({
+                    open: true,
+                    data: {
+                        header: 'View Comments',
+                        subText: [
+                            {
```

Comment:
I feel I'm strongly against having pop ups for this purpose. We can have a ticket to display text in the table when list is length 0. I appreciate if we could hold off merging this until we discuss it at least with MA

Reply:
made the changes